src/Message.php

```diff
@@ -44,7 +44,7 @@ public static function fromRaw($body)
 
     public function getBody($part = 'html')
     {
-        return Arr::get($this->view, $part, Arr::first($this->view));
+        return Arr::get($this->view, $part, reset($this->view));
```

Comment:
As it seems to me, since it's not being passed with a closure, it's just expecting to get the first element of the `$this->view` array. There's no default and no filter. As such, PHP's native `reset` seems perfectly fine to me. :+1: